### PR TITLE
Rename all the Polygon Hermez references to Polygon zkEVM

### LINKED ISSUE #129: Update project branding
We need to update the project branding to `Polygon zkEVM` instead of `Polygon Hermez`.

### PR DESCRIPTION
**Pull request by elias-garcia**
Closes #129.

### What does this PR does?

This PR covers:

- Rebranding from `Polygon Hermez` to `Polygon zkEVM`.
- Improvement of the `README.md` file.
- Fixes the styles of `Header` component.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [x] Update documentation (if needed)

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const getChains = ({
         explorerUrl: ethereum.explorerUrl,
       },
       {
-        key: "polygon-hermez",
+        key: "polygon-zkevm",
         networkId: polygonHermez.networkId,
```
[the remark this answers is not in the document]
Comment (elias-garcia) [reply src/constants.ts]: Right! Done

File: README.md [hunk 2]
```diff
@@ -22,30 +22,40 @@ Install project dependencies:
...
+If you want to connect to some knwon environment, we provide three prefilled `.env` files, which you can also use to create your `.env` file:
 
-## License
-
-`zkevm-bridge-ui` is part of the Polygon Hermez project copyright 2022 HermezDAO and published with AGPL-3 license. Please check the LICENSE file for more details.
+- `.env.integration`: Locally deployed zkEVM.
+- `.env.internal-testnet`: Official internal testnet, which is used to do the official demos.
+- `.env.fetestnet`:  Internal testnet used by us, the zkEVM Bridge UI team to speed up the development.
 
+Take into account that regardless the option you choose, you will need to manually add the `REACT_APP_FIAT_EXCHANGE_RATES_API_KEY` environment variable, which you can obtain [here](https://exchangeratesapi.io/). If you omit it, you won't be able to see token prices converted to your local fiat currency.
```
[the remark this answers is not in the document]
Comment (elias-garcia) [reply README.md]: Fixed!

File: README.md [hunk 3]
```diff
@@ -22,30 +22,40 @@ Install project dependencies:
...
-Create the required `.env` file from the example provided in the repo:
+Finally, to be able to run the project, you need to create a `.env` file which should contain all the required environment variables.
+
+If you want to create it from scratch, you can copy the `.env.example` and then override each environment variable by your own:
 
 ```sh
 cp .env.example .env
 ```
 
+If you want to connect to some knwon environment, we provide three prefilled `.env` files, which you can also use to create your `.env` file:
```
[the remark this answers is not in the document]
Comment (elias-garcia) [reply README.md]: Fixed!

File: src/constants.ts [hunk 4]
```diff
@@ -45,24 +45,24 @@ export const ETH_TOKEN_LOGO_URI =
 
 export const getChains = ({
   ethereum,
-  polygonHermez,
+  polygonZkEVM: polygonZkEVM,
```
Comment (amonsosanz) [inline src/constants.ts:48]: I'd remove the key, as we do above

**Review by amonsosanz — APPROVED** [review]

File: src/constants.ts (hunk 4 above)
Comment (elias-garcia) [reply src/constants.ts]: Done!

**Pull request merged**